### DISCUSSION TITLE
04/064-%E5%B9%95%E9%97%B4-3-%E6%98%AD%E6%98%AD%E5%A4%A9%E5%91%BD.html

### DISCUSSION THREAD

**Discussion by giscus** [liuli-moe/to-the-stars#19] [General]
04/064-%E5%B9%95%E9%97%B4-3-%E6%98%AD%E6%98%AD%E5%A4%A9%E5%91%BD.html
在经历了几个世纪的动荡之后，一个乌托邦式的 AI— 人类政府治理着地球，预示着后稀缺社会的来临和太空殖民的新时代。一次意外的接触却让科技更先进的敌对外星种族打破了和平，这迫使魔法少女们走出幕后，拯救人类文明。在这一切之中，志筑良子，一个普通的女孩，仰望着星空，好奇着她在宇宙中的归所。
https://tts.liuli.moe/04/064-%E5%B9%95%E9%97%B4-3-%E6%98%AD%E6%98%AD%E5%A4%A9%E5%91%BD.html

**Comment by ArgusK17** [comment]
本章尾声似乎是读者首次站在焰的视角，解开了许多之前的困惑（特别是发现是圆神竟然能打私人电话2333）
另外每次剧情冲击力不够就虐焰真的好吗x
